feat: update TPC banner image

preda-bogdan opened the pull request:
### Summary
<!-- Please describe the changes you made. -->
Updated the image for the TPC banner.

### Will affect the visual aspect of the product
<!-- It includes visual changes? -->
YES

### Screenshots <!-- if applicable -->
![image](https://user-images.githubusercontent.com/23024731/234254977-39b6dc81-843f-4d8d-aa19-32f11ce07fb3.png)


### Test instructions
<!-- Describe how this pull request can be tested. -->
1. Create a new instance with Neve
2. Check inside the dashboard that the banner from the screenshot ☝🏼 has the updated image.

## Check before Pull Request is ready:

* [ ] I have [written a test](CONTRIBUTING.md#writing-an-acceptance-test) and included it in this PR
* [x] I have [run all tests](CONTRIBUTING.md#run-tests) and they pass
* [x] The code passes when [running the PHP CodeSniffer](CONTRIBUTING.md#run-php-codesniffer)
* [x] Code meets [WordPress Coding Standards](CONTRIBUTING.md#coding-standards) for PHP, HTML, CSS and JS
* [x] [Security and Sanitization](CONTRIBUTING.md#security-and-sanitization) requirements have been followed
* [x] I have assigned a reviewer or two to review this PR (if you're not sure who to assign, we can do this step for you)

<!-- Issues that this pull request closes. -->
Closes #3919.
<!-- Should look like this: `Closes #1, #2, #3.` . -->

irinelenache commented:
@preda-bogdan Tested and the screenshot is updated, but for me it seems a bit blurrier than the previous image:

How it looked [before](https://vertis.d.pr/i/kxsyXH) vs how it looks [now](https://vertis.d.pr/i/FqSZCl)
Inspected image [before](https://vertis.d.pr/i/1s1j8E) vs inspected image [now](https://vertis.d.pr/i/wN7x1j)

You can check here:
```
https://obfxphp.vertisite.cloud/wp-admin
User: liwotixaha9326
Password: GHC7VXtvQ9ya0oKD4ObU
```

preda-bogdan commented:
@mghenciu I took the image from the Figma file, do we have a better resolution of that image?

mghenciu commented:
I've made one 2x. Let me know if it's better.
[Archive.zip](https://github.com/Codeinwp/neve/files/11323226/Archive.zip)

You can export (if without edit rights) different sizes for a component in Figma:
![Screenshot 2023-04-25 at 16 54 37](https://user-images.githubusercontent.com/52494172/234299507-eb7c94c0-f041-4bad-ba38-bfe5c41b61c9.png)

preda-bogdan commented:
@irinelenache I've updated the image with a higher resolution one.
@mghenciu thank you, I exported it directly without opening the modal so it was at the display size of 200px. I've fixed it now.
